spec/extensions/pretty_print_spec.rb

```diff
@@ -1,54 +1,47 @@
 # frozen_string_literal: true
 
 RSpec.describe Dry::Struct do
-  describe "#pretty_print" do
+  describe "#pretty_inspect" do
```

Comment:
@pyromaniac I guess it should be either `describe "#pretty_print"` or `describe "pretty printing"`, am I right? `describe "#foo"` is a convention for describing the `foo` method. Pls confirm and we can have it merged

Reply:
We are literally checking how `#pretty_inspect` method works here cause this is a public api and the direct consequence of the `pretty_print` method implementation which is a private api.

Reply:
I see, thansk